### DISCUSSION TITLE
POLLS

### DISCUSSION THREAD

**Discussion by TT-ReBORN** [TT-ReBORN/Georgia-ReBORN#96] [General]
## Everyone can participate in those community based polls:

[What is your favorite theme ?](https://poll-maker.com/poll3973678xb0d456fF-126)

[Which themes are you using ( multiple votes allowed ) ?](https://poll-maker.com/poll3973724x2c3c1C85-126)

[What are your favorite theme style presets ( multiple votes allowed ) ?](https://take.quiz-maker.com/poll4518813x45d74ccB-141)

[Which are your favorite disc art placeholders ( multiple votes allowed ) ?](https://poll-maker.com/poll3973713x80B14a21-126)

[Which layouts are you using ( multiple votes allowed ) ?](https://poll-maker.com/poll3973702x5AffCf2e-126)

[Are you using and creating playlists ?](https://poll-maker.com/poll3973750x1a5743F3-126)

[Which playlist album date order do you prefer ?](https://poll-maker.com/QOB5A1U72)

[What music genres do you listen to ( multiple votes allowed ) ?](https://poll-maker.com/poll3973844x28F34BB6-126)

---

**Thank you!**

**P.S: You can also post your own polls in this thread here!**

-TT

**Comment by d3rb** [comment]
[What music genres do you listen to ( multiple votes allowed ) ?](https://poll-maker.com/poll3973844x28F34BB6-126)

Jazz
Vocal Jazz
Jazz Rock
New Age
Country
Blues
Soul
Folk

**Comment by Zephyr0ck** [comment]
What is your favorite Georgia-ReBORN theme?

- Reborn

Which themes are you using ( multiple votes allowed ) ?

- Reborn

What are your favorite theme style presets ? ( Multiple votes allowed! )

- Reborn > Reborn black blended
- Reborn > Reborn black blended 2

Which are your favorite Georgia-ReBORN disc art placeholders? ( Multiple votes allowed! )

- My own customized Vinyl 😋 

Which Georgia-ReBORN layouts are you using? ( Multiple votes allowed! )

- Default

Are you using and creating playlists?

- No

Which playlist album date order do you prefer?

- Ascending order ( oldest to latest ( top to bottom ) = 2001, 2002, 2003 ... 2022 )

What music genres do you listen to? ( Multiple votes allowed! )

- Metal (Black, Death, Symphonic, Melodic, Depressive, Slam Brutal)
- Jazz
- Rap, Hip Hop


✌️

**Comment by TT-ReBORN** [comment]
Thanks guys for participating, but did you guys vote? Just to be sure, in the first post are links where you can vote ;-)

**Reply by Zephyr0ck** [reply]
Yep, I did it ;)
